docs/src/pages/getting-started/example-projects/example-projects-de.md

```diff
@@ -23,15 +23,23 @@ Suchen Sie nach einem fortgeschritteneren Beispielprojekt?
 If you want to start with a more complete and real world example, you could take a look at our [free themes](https://themes.material-ui.com/) or:
 
 - [React + Material-UI + Firebase](https://github.com/Phoqe/react-material-ui-firebase):
-  
+
   - Mit Create React App gebootet, funktioniert das gleiche Tool sofort
   - Aufbauend auf Firebase mit Authentifizierung von Anfang an
   - Robustes Routing mit React Router inklusive Fehlerbehandlung (404)
   - Umfassender mobiler Support mit [react-swipeable-views](https://react-swipeable-views.com) für Tabs
 
 - [Material Sense](https://github.com/alexanmtz/material-sense), welches beeinhaltet:
-  
+
   - Diagramme mit recharts
   - React Router mit einem Navigationsbeispiel
   - Ein Docker-Container mit einem Nginx-Server für den Produktionsaufbau
-  - Erstellt mit [Create React App](https://facebook.github.io/create-react-app/)
\ No newline at end of file
+  - Erstellt mit [Create React App](https://facebook.github.io/create-react-app/)
+
+- [React Most Wanted](https://github.com/TarikHuber/react-most-wanted), which includes:
```

Comment:
Could you revert the changes to the translated markdowns? They shouldn't be touched.